### PR TITLE
Add http file server tutorial.

### PR DESCRIPTION
**Pull request by floitsch**

### CODE REVIEW

File: docs/tutorials/network/http-file-server.mdx [hunk 1]
```diff
@@ -0,0 +1,251 @@
+# HTTP
+In this tutorial we will write an HTTP file server. The server
+takes the HTTP path and serves the corresponding file from the
+local file system. In spirit it is very similar to
+python's `python -m http.server` command.
+
+We will learn how to use the `http` package to serve HTML pages and the
```
Comment (kasperl) [inline docs/tutorials/network/http-file-server.mdx:7]:
```suggestion
You will learn how to use the `http` package to serve HTML pages and the
```

File: tutorial_code/http-file-server/file-server.toit [hunk 2]
```diff
@@ -0,0 +1,122 @@
+import host.file
```
Comment (kasperl) [inline tutorial_code/http-file-server/file-server.toit:1]: Maybe call this `file_server.toit` for now?

**Review by kasperl — APPROVED** [review]
Looks good to me.

File: docs/tutorials/network/http-file-server.mdx (hunk 1 above)
Comment (floitsch) [reply docs/tutorials/network/http-file-server.mdx]: done.

File: tutorial_code/http-file-server/file-server.toit (hunk 2 above)
Comment (floitsch) [reply tutorial_code/http-file-server/file-server.toit]: done.

**Pull request merged**